doc/api/deprecations.md

```diff
@@ -752,7 +752,7 @@ changes:
 Type: Documentation-only
 
 The [`events.listenerCount(emitter, eventName)`][] API is
-deprecated. Please use [`emitter.listenerCount(eventName)`][] instead.
+deprecated. Please use [`emitter.listenerCount(eventName, listener)`][] instead.
```

Comment:
IMO the correct replacement for `events.listenerCount(emitter, eventName)` is `emitter.listenerCount(eventName)`, not `emitter.listenerCount(eventName, listener)`.
```suggestion
deprecated. Please use [`emitter.listenerCount(eventName)`][] instead.
```